nova/app.py

```diff
@@ -103,8 +106,12 @@ def extract_data(image):
             extracted_value2 = float(extracted_text)
             extracted_data[idx + 1] = f"{keys[idx]} {extracted_value2:.2f}"
 
-    new_image = f"{content_type},{content_string}"
-    return extracted_data, new_image
```

Comment:
This was a bad practice. I wasn't following the principle of "separation of concers".

It is rarely a good idea to return tuples from functions. Every time that happens you can ask yourself

"Am I violating the principle of separation of concerns?"